deployment/helm/polder/values.yaml

```diff
@@ -0,0 +1,15 @@
+# Default values for polder.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This is to override the chart name.
+nameOverride: "polder"
+fullnameOverride: "test-polder"
+
+# polder custom configuration
+mapboxToken: "pk.eyJ0IjoiZGV2c2VlZCIsImEiOiJnUi1mbkVvIn0.018aLhX0Mb0tdtaT2QNe2Q"
```

Comment:
Whatever this token is. We should definitely remove it *and* regenerate it on Mapbox... Given this repo is public, I assume this is now exposed and vulnerable

Reply:
Oh, I am sorry. I was assuming this was just to be some dummy token.

Shall we handle normal k8s secrets instead? And share the file in some private repo.